taca/cleanup/cleanup.py

```diff
@@ -132,14 +134,18 @@ def cleanup_irma(days_fastq, days_analysis, only_fastq, only_analysis, clean_und
         if date:
             date = datetime.strptime(date, '%Y-%m-%d')
     except KeyError as e:
-        logger.error('Config file is missing the key {}, make sure it have all required information'.format(str(e)))
+        logger.error('Config file is missing the key {}, make sure it has all required information'.format(str(e)))
         raise SystemExit
     except ValueError as e:
         logger.error('Date given with "--date" option is not in required format, see help for more info')
         raise SystemExit
 
-    # make a connection for project db #
-    pcon = statusdb.ProjectSummaryConnection(conf=status_db_config)
+    # make a connection for project db
+    config_statusdb = CONFIG.get('statusdb', None)
```

Comment:
This wouldn't work with our current setup. Probably go back to using the env var here?

Reply:
Reverted to having a separate config for statusdb credentials